cmd/scollector/collectors/aws.go

```diff
@@ -0,0 +1,302 @@
+package collectors
+
+import (
+	"fmt"
+	"time"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+	"bosun.org/slog"
+
+	"github.com/awslabs/aws-sdk-go/aws"
+	"github.com/awslabs/aws-sdk-go/gen/cloudwatch"
+	"github.com/awslabs/aws-sdk-go/gen/ec2"
+	"github.com/awslabs/aws-sdk-go/gen/elb"
+)
+
+const (
+	awsCPU               string = "aws.ec2.cpu"
+	awsNetwork           string = "aws.ec2.net.bytes"
+	awsDiskBytes         string = "aws.ec2.disk.bytes"
+	awsDiskOps           string = "aws.ec2.disk.ops"
+	awsStatusCheckFailed string = "aws.ec2.status.failed"
+	awsELBLatency        string = "aws.elb.latency"
+	awsELBHostsHealthy   string = "aws.elb.hosts.healthy"
+	awsELBHostsUnHealthy string = "aws.elb.hosts.unhealthy"
+)
+
+func AWS(accessKey, secretKey, region string) {
+	collectors = append(collectors, &IntervalCollector{
+		F: func() (opentsdb.MultiDataPoint, error) {
+			return c_aws(accessKey, secretKey, region)
+		},
+		Interval: 60 * time.Second,
+		name:     fmt.Sprintf("aws-%s", region),
+	})
+}
+
+func c_aws(accessKey, secretKey, region string) (opentsdb.MultiDataPoint, error) {
+	var md opentsdb.MultiDataPoint
+	creds := aws.Creds(accessKey, secretKey, "")
+	if creds == nil {
+		slog.Info("Unable to make creds")
```

Comment:
fixed in 42b874b03705295d53ad9363d4317e6dae32a124 and ddc4da550b66f0be113ddb0f088b12f6dbfebf86.